test/cppsuite/configs/search_near_01_stress.txt

```diff
@@ -0,0 +1,22 @@
+# Configuration for search_near_01 stress test.
+# The test will generate key_count_per_collection number of keys for each prefix in aaa -> zzz.
+# This config will have a 30 minute duration, with 3 tables of an entry for each prefix.
+duration_seconds=1800,
+cache_size_mb=1000,
+timestamp_manager=
+(
+    enabled=false,
+),
+workload_generator=
+(
+    populate_config=
+    (
+        collection_count=3,
+        key_count_per_collection=100,
```

Comment:
@jiechenbo , I had a look at the code and this does not seem to be used at all. We should remove it from both cfgs?

Reply:
If this setting is not used, we should remove it from the code.